lib/fog/ovirt/compute/v4.rb

```diff
@@ -100,6 +100,8 @@ def get_attr_value(value, opts)
             else
               if value.class.respond_to?(:parent) && value.class.parent == OvirtSDK4
                 value.id if value.respond_to?(:id)
+              elsif [true, false].include? value
```

Comment:
Wouldn't it be a bit nicer to add `TrueClass, FalseClass` to [line 79](https://github.com/fog/fog-ovirt/pull/41/files#diff-22bad1e3c6fc8e9e1c55d918dd9c1f0aL79)?